ably/rest/rest.py

```diff
@@ -65,6 +65,10 @@ def __init__(self, key=None, token=None, token_details=None, **kwargs):
         # else:
         #     self.__session = None
 
+        self.__api_version = "0.9"
+        self.__api_lib = "0.9.0"
+        self.__lib_variant = "ably-python-"
```

Comment:
This variant is not according to the spec, see http://docs.ably.io/client-lib-development-guide/features/#RSC7b.  It states `The header X-Ably-Lib: [lib][.optional variant]?-[version] should be included in all REST requests to the Ably endpoint where [lib] is the name of the library such as js for ably-js, [.optional variant] is an optional library variant, such as laravel for the php library, which is always delimited with a period such as php.laravel`.  So firstly the string should be `python` not `ably-python.` Secondly, the variant is defined so that later we could perhaps have a variant for `twisted` and define that making it `python.twisted`.